fjord/feedback/tests/test_views.py

```diff
@@ -115,68 +117,60 @@ def test_valid_sad(self):
         eq_(u'14.0.1', feedback.version)
 
     @with_waffle('thankyou', True)
-    def test_use_thank_you_sad(self):
-        """Thanks_sad template is used when appropriate"""
+    def test_thankyou_flag_active(self):
+        """Verify response and suggestions when thankyou flag is active"""
         url = reverse('feedback', args=(u'firefox',), locale='en-US')
         r = self.client.post(url, {
             'happy': 0,
-            'description': u"I want to know why Firefox doesn't make me sandwiches!",
+            'description': u'Why Firefox not make me sandwiches!',
```

Comment:
So, the impetus behind the change was the long line. However, I made the text like this because it's far more like real input and I thought it was kind of funny. I wanted to keep the general sentiment Adam had, too. It refers to the feedback description in another test in this file.

Another iteration I was toying with was "Y U NO MAKE SANDWICHES!!!!!" and then put enough ! to fill 79 characters. After seeing it in the code, I decided it was too horrible to force my future self or others to read.